test/sequential/test-deprecation-flags.js

```diff
@@ -16,8 +16,9 @@ execFile(node, normal, function(er, stdout, stderr) {
   console.error('normal: show deprecation warning');
   assert.equal(er, null);
   assert.equal(stdout, '');
-  assert.equal(stderr, '(node) util.debug is deprecated. Use console.error ' +
-                       'instead.\nDEBUG: This is deprecated\n');
+  const stderrResult = stderr.replace(/ *\([^)]*\) */g, '');
+  assert.equal(stderrResult, 'util.debug is deprecated. Use console.error ' +
```

Comment:
Sorry, just looking at this again. Couldn't you just `assert(/util\.debug is deprecated.../.test(stderr));`